src/packaging/metadata.py

```diff
@@ -749,8 +752,9 @@ def from_email(
         try:
             return cls.from_raw(raw, validate=validate)
         except ExceptionGroup as exc_group:
-            exceptions.extend(exc_group.exceptions)
```

Comment:
This was never going to do anything, because `exceptions` was always going to be an empty list, and it was making my editor sad because `ExceptionGroup.exceptions` has an incompatible (with `list[InvalidMetdata]`) type.

Reply:
Huh, guess fixing this also made MyPy just generally unhappy.